tests/robottelo/test_func_locker.py

```diff
@@ -0,0 +1,321 @@
+# coding: utf-8
```

Comment:
Think they are synonyms https://www.python.org/dev/peps/pep-0263/
and also # -*- coding: utf-8 -*-
if you know any specific diff please share